atomicapp/utils.py

```diff
@@ -372,6 +373,11 @@ def copy_dir(src, dest, update=False, dryrun=False):
             distutils.dir_util.copy_tree(src, dest, update)
 
     @staticmethod
+    def copy_file(src, dest, dryrun=False):
```

Comment:
did you forget to add a file to the commit? I don't see where copy_file gets used anywhere.